p2panda-net-next/Cargo.toml

```diff
@@ -8,13 +8,15 @@ workspace = true
 
 [dependencies]
 ciborium = "0.2.2"
+futures = "0.3.31"
```

Comment:
I believe everything you need from `futures` is already in `futures-util`. In the test environment you might want to use `futures-channel`.

Reply:
I'm using an mpsc channel from futures in both sync manager and session actors.